docs/npwd/Credits.md

```diff
@@ -4,21 +4,22 @@ title: Contributers
 sidebar_label: Contributers
 ---
 
-We appreciate all the support and help we get. Looking forward to making this list longer!
+We appreciate all the support and help we get. Below is a list of contributors to the resource with the [**Project Error Team**](https://github.com/orgs/project-error/people) removed from the list. We hope we continuously add to this list as time passes.
 
-- [**Oldarorn**](https://github.com/Oldarorn)
 - [**AvarianKnight**](https://github.com/AvarianKnight)
 - [**BombayV**](https://github.com/BombayV)
-- [**synterrr**](https://github.com/synterrr)
-- [**dayiskuan**](https://github.com/dayiskuan)
-- [**brezedc**](https://github.com/brezedc)
-- [**BTNGaming**](https://github.com/BTNGaming)
```

Comment:
I removed/ didnt include contributors who made irrelevant comments like [licenses](https://github.com/project-error/npwd/commits?author=BTNGaming) and [print removal](https://github.com/project-error/npwd/commit/42830ecdace5647175df1a674df0fb30382cbcdc)

Reply:
Fair enough